Dockerfile.windows

```diff
@@ -0,0 +1,15 @@
+# escape=`
+
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
+
+ARG PIXI_VERSION
+
+# Create a directory for Pixi
+RUN mkdir C:\Pixi
+
+# Download Pixi
+ADD https://github.com/prefix-dev/pixi/releases/download/v${PIXI_VERSION}/pixi-x86_64-pc-windows-msvc.exe C:\Pixi\pixi.exe
```

Comment:
while we're at it: there is also `https://github.com/prefix-dev/pixi/releases/download/v0.39.0/pixi-aarch64-pc-windows-msvc.exe`. wdyt of doing a multi-arch build on windows as well similar to linux?